code-reviews.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Code Reviews at OfficeLuv
+layout: default
+permalink: '/code-reviews'
+---
+
+## Why Do Code Reviews?
+
+Doing code reviews:
+
+- shares knowledge of the codebase and business
+- catches bugs automated testing may not catch
+- uncovers blindspots we may have about the codebase
+- improves overall code quality and health
+- improves our communication skills
+
+## How OfficeLuv Does Code Reviews
+
+We do code reviews once a day, all together, in person. Doing them this way makes knowledge sharing more efficient. Async reviews also can be requested throughout the day for any work that's blocking or more urgent.
+
+Each dev presents their code by briefly explaining the problem and their overall solution. The dev will walk the group through their code, pointing out anything they like or dislike about it. Other devs then ask questions and may give advice on different approaches. If there's nothing blocking the pull request at the end of this process, the code is merged.
+
+Every dev participates in the code review process the same way, regardless of seniority.
+
```

Comment:
```suggestion
Code Review is also a time reserved for reviewing external code. This can be open-source work members are currently working on, something they were proud of cracking, or just an interesting link or library.
We also review applicant code samples as a group during our Code Review sessions - just as we would review our own code.

```